Makefile

```
@@ -0,0 +1,77 @@
+.PHONY: run test build clean docker-up docker-down docker-build docker-logs docker-logs-backend docker-logs-frontend docker-mysql lint
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)